tests/parametric/test_128_bit_traceids.py

```diff
@@ -193,7 +193,6 @@ def test_datadog_128_bit_generation_enabled(self, test_agent, test_library):
 
     @missing_feature(context.library == "golang", reason="not implemented")
     @missing_feature(context.library < "java@1.24.0", reason="Implemented in 1.24.0")
-    @missing_feature(context.library == "nodejs", reason="not implemented")
     @missing_feature(context.library == "ruby", reason="not implemented")
     @pytest.mark.parametrize("library_env", [{"DD_TRACE_PROPAGATION_STYLE": "Datadog"}])
     def test_datadog_128_bit_generation_enabled_by_default(self, test_agent, test_library):
```

Comment:
would be nice to have this be a version check instead of removing the whole line, just like java is doing right above.
(same thing in line 412)